pages/index.js

```diff
@@ -1,18 +1,36 @@
 import React from "react";
+import Head from "next/head";
 import { Normalize } from "styled-normalize";
+import readConfig from "../scripts/read-config";
 import { GlobalStyle } from "../src/common/GlobalStyle/GlobalStyle";
 import App from "../src/components/App/App";
 import { ThemeProvider } from "../src/contexts/Theme";
 import { ConfigProvider } from "../src/contexts/Config";
 
-const indexPage = () => (
-  <ThemeProvider>
-    <Normalize />
-    <GlobalStyle />
-    <ConfigProvider>
-      <App />
-    </ConfigProvider>
-  </ThemeProvider>
+const indexPage = ({ config }) => (
+  <>
+    <Head>
+      <title>title</title>
+    </Head>
+    <ThemeProvider>
+      <Normalize />
+      <GlobalStyle />
+      <ConfigProvider config={config}>
```

Comment:
Perhaps it's weird to pass something to a context provider but it's required as `getStaticProps()` is limited for pages: https://nextjs.org/docs/basic-features/data-fetching#only-allowed-in-a-page (also left this as a comment below).